src/3rdparty/walkontable/src/renderer/columnHeaders.js

```diff
@@ -77,6 +77,10 @@ export default class ColumnHeadersRenderer extends BaseRenderer {
         TH.removeAttribute('style');
 
         columnHeaderFunctions[visibleRowIndex](sourceColumnIndex, TH, visibleRowIndex);
+
+        if (sourceColumnIndex < 0) {
+          TH.classList.add('wtTableCornerCell');
```

Comment:
The `classList` feature is not supported by IE9. This is intentional?

Reply:
Thanks for noticing. I keep doing this by mistake. Maybe an ESLint plugin would save me ;) https://github.com/amilajack/eslint-plugin-compat